source/api_documentation/network_integration/call_activity_tracking/index.rst

```diff
@@ -0,0 +1,51 @@
+Call Activity Tracking
+======================
+
+Manage call activity tracking for campaigns
+"""""""""""""""""""""""""""""""""""""""""""
+
+Call activity tracking can be set for a campaign to determine whether or not call activity tracking should be enabled. The call activity will be tracked if the full condition is met.
+
+Set Call Activity Tracking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To conditionally enable call activity tracking, set campaign's owning network `call_activity_access` feature flag to `true`, and set ``"call_activity_tracking"`` to ``true``.
```

Comment:
Via the API, the users will not be able to enable the `calll_activity_access` feature flag so lets take that out here. Additionally, `call_activity_tracking` provides multiple options and is not a boolean
```
To conditionally track call activity, set ``"call_activity_tracking"`` to the tracking option you want.
```

Reply:
Fixed, thank you!